aphrodite/modeling/layers/sampler.py

```diff
@@ -318,12 +318,18 @@ def _apply_temperatures(
     normalized_entropies = dynatemp_entropies.div_(dynatemp_max_entropies)
```

Comment:
Unrelated to the PR, but I wonder if there might be cases where `dynatemp_max_entropies` would contain zeros (possible if all logits are -inf). This would end up producing NaNs.

Reply:
I believe it can, though practically speaking I would only expect this to be an issue with temperature_last.

(just tested, yes, yes, it can. easily fixed, though.)